ddl/schema.go

```diff
@@ -79,12 +121,16 @@ func onDropSchema(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 		job.State = model.JobStateCancelled
 		return ver, infoschema.ErrDatabaseDropExists.GenWithStackByArgs("")
 	}
+	// Rolling back drop schema.
+	if job.IsRollingback() && job.Type == model.ActionDropSchema {
```

Comment:
Why do you add this judgment here `job.Type == model.ActionDropSchema`?

Reply:
Maybe job.Type == model.ActionDropSchema is redundant?